### ISSUE TITLE
Elements in shadowRoot are not directly supported

### ISSUE THREAD

**Issue by PirogovClub** [healenium/healenium-web#73]
While all WebElements allocated with driver.findelement working fine, WebElements that was casted from JavaScript execution obviously are not wrapped with WebElementProxyHandler

Example for obtaining shadowRootObject
```
private static WebElement getShadowRoot(WebDriver driver,WebElement shadowHost) {
		JavascriptExecutor js = (JavascriptExecutor) driver;
		return (WebElement) js.executeScript("return arguments[0].shadowRoot", shadowHost);
	}
```

Can be method for wrapping those WebElements be added to the SelfHealingDriver?
Or additional handler need to be added to JavaScriptExecutor event.

Or, is there any other naitive way to use tool for working with ShadowRoots?

**Comment by Aliaksei-Ashukha** [comment]
Hi @PirogovClub 
Sure, we'll add wrapping for these WebElements.
Thanks for your сomment

**Comment by PirogovClub** [comment]
> Hi @PirogovClub
> Sure, we'll add wrapping for these WebElements.
> Thanks for your сomment

Hey,

I actually add that if you wanna look. It is in the fork of you project.
I can brush it a bit and create a pull request, if you are ok with that.
https://github.com/timofeypirogov-accenture/healenium-web/tree/feature/addShadowRootSupport

Br, Tim

**Comment by GannaChernyshova** [comment]
Hi @PirogovClub! That's great! We'll be happy to have a pull request from you

**Comment by GannaChernyshova** [comment]
fixed in version 3.1
